src/pages/Travel/TravelStatusBus.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import {
+  BUS_SHOPIFY_PAGE,
+  BUS_SLACK_CHANNEL,
+  BUS_SLACK_PAGE,
+  ITravel,
+} from '../../config';
+import { LinkDuo } from '../../shared/Elements';
+
+interface ITravelStatusProps {
+  travel: ITravel;
+}
+
+const TravelStatusBus: React.FunctionComponent<ITravelStatusProps> = ({
+  travel,
+}) => {
+  return (
+    <div>
+      Congratulations, you've secured a seat on our Toronto bus to/from McHacks!
```

Comment:
nit: "Congratulations, you've secured a seat on our {BUS_DEPARTURE_CITY} bus to/from McHacks!"

Reply:
Done